packages/components/src/internal/components/search/SearchResultCard.tsx

```diff
@@ -61,17 +61,13 @@ export const SearchResultCard: FC<SearchResultProps> = memo(({ cardData, iconUrl
                         />
                     )}
                 </div>
-
-                <div className="col-sm-10">
-                    <div>
+                <div className="col-sm-12">
+                    <div className="col-sm-12 search-result__title">
                         <h4 className="text-capitalize">{title}</h4>
+                        {category && <div className="search-result__category-badge">{category}</div>}
+                        {typeName && <div className="search-result__type-badge">{typeName}</div>}
```

Comment:
I made each of these (Type and Category) a badge as trying to force them into a single badge led to all kinds of odd grammar. This seemed more natural.